keywords/couchbaseserver.py

```diff
@@ -327,7 +330,7 @@ def get_ram_per_bucket(self, num_buckets):
         ram_per_bucket_mb = int(effective_ram_mb / num_buckets)
         return ram_per_bucket_mb
 
-    def create_buckets(self, bucket_names, ipv6=False):
+    def create_buckets(self, bucket_names, cluster_config, ipv6=False):
```

Comment:
This method is used in install_sync_gateway.py and cluster.py which does not pass cluster_config. you can pass default value, so that it won't effect those files.

Reply:
I've made changes in those file to pass cluster config.